README.md

```diff
@@ -133,7 +133,18 @@ The package is tested with [PHPUnit](https://phpunit.de/). For tests need Postgr
 - port: `5432`
 - name: `testdb`
 - user: `postgres`
-- password: `postgres` 
+- password: `postgres`
+
+#### Docker Image
+
+To easily set up a pre-configured PostgreSQL instance for testing you can use the [docker-compose.yml](https://docs.docker.com/compose/compose-file/) file that is in the `/tests/Support/docker/pgsql` folder.
+[PostgreSQL 15](/tests/Support/docker/pgsql/docker-compose.yml)
```

Comment:
Please don't store docker files inside subdirectories.
It's a package. Any configuration files in the root directory are alright.

Reply:
@xepozz OK, I moved the docker-compose.yml to the root, also updated the Readme.